lib/portlayer/exec/exec.go

```diff
@@ -84,9 +84,6 @@ func Init(ctx context.Context, sess *session.Session, source extraconfig.DataSou
 		// create the event manager &  register the existing collector
 		Config.EventManager = event.NewEventManager(ec)
 
-		// subscribe the exec layer to the event stream for Vm events
-		Config.EventManager.Subscribe(events.NewEventType(vsphere.VMEvent{}).Topic(), "exec", eventCallback)
```

Comment:
Without this we are not going to get events from vsphere about out of band power offs, etc.